action.yml

```
@@ -6,9 +6,15 @@ description: Setup a Zephyr base project using west and downloading the Zephyr S
 
 inputs:
   app-path:
-    description: Application code path, should contain a west.yml file
+    description: |
+      Application code path, should contain a west.yml file if `manifest-file-name` is not specified
```

```suggestion
      Application code path, should contain a west.yml file if "manifest-file-name" is not specified
```

Just to be coherent with the quote character used in sdk-version